jenkins-slave

```diff
@@ -75,8 +75,20 @@ else
 		fi
 	fi
 
+	# JENKINS_CA_CERT points a path for ca cert (possibly mounted as a secret volume)
+	if [ -n "$JENKINS_CA_CERT" ]; then
+		jre_keystore=/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/security/cacerts
+		keystore_dir=$HOME/.keystore
+		mkdir -p $keystore_dir
+		cp $jre_keystore $keystore_dir
+		cat $JENKINS_CA_CERT > /tmp/cacert
+		keytool -import -keystore $keystore_dir/cacerts -alias master -file /tmp/cacert -storepass changeit -noprompt
+		rm /tmp/cacert
+		TRUSTSTORE="-Djavax.net.ssl.trustStore=$HOME/.keystore/cacerts -Djavax.net.ssl.trustStorePassword=changeit"
+	fi
+
 	#TODO: Handle the case when the command-line and Environment variable contain different values.
 	#It is fine it blows up for now since it should lead to an error anyway.
 
-	exec java $JAVA_OPTS $JNLP_PROTOCOL_OPTS -cp /usr/share/jenkins/slave.jar hudson.remoting.jnlp.Main -headless $TUNNEL $URL $OPT_JENKINS_SECRET $OPT_JENKINS_AGENT_NAME "$@"
+	exec java $JAVA_OPTS $TRUSTSTORE $JNLP_PROTOCOL_OPTS -cp /usr/share/jenkins/slave.jar hudson.remoting.jnlp.Main -headless $TUNNEL $URL $OPT_JENKINS_SECRET $OPT_JENKINS_AGENT_NAME "$@"
```

Comment:
Do you have any example? Because user jobs can do that today? This was the recommended approach in https://support.cloudbees.com/hc/en-us/articles/203821254-How-to-install-a-new-SSL-certificate